src/pages/CommitDetailPage/CommitCoverage/UploadsCard/useUploads/useUploads.js

```diff
@@ -3,20 +3,15 @@ import { useParams } from 'react-router-dom'
 import { useCommit } from 'services/commit'
 import { useRepoOverview } from 'services/repo'
 import { TierNames, useTier } from 'services/tier'
-import { useFlags } from 'shared/featureFlags'
 import { extractUploads } from 'shared/utils/extractUploads'
 
 export function useUploads() {
   const { provider, owner, repo, commit } = useParams()
   const { data: overview } = useRepoOverview({ provider, owner, repo })
-  const { multipleTiers } = useFlags({
-    multipleTiers: false,
-  })
   const { data: tierData } = useTier({ provider, owner })
 
   // TODO: We need backend functionality to properly manage access to carryforward flags for team tier members
```

Comment:
Is this TODO now obsolete? TBH I have no idea what it's trying to say :joy:

Reply:
im right there with ya loool, that's a TODO to figure out later 😂